hcatalog/core/src/main/java/org/apache/hive/hcatalog/cli/SemanticAnalysis/HCatSemanticAnalyzer.java

```diff
@@ -336,7 +336,7 @@ protected void authorizeDDLWork(HiveSemanticAnalyzerHookContext cntxt, Hive hive
     } else if (ddlDesc instanceof AlterTableSetLocationDesc) {
       AlterTableSetLocationDesc alterTable = (AlterTableSetLocationDesc)ddlDesc;
       Table table = hive.getTable(SessionState.get().getCurrentDatabase(),
-          Utilities.getDbTableName(alterTable.getTableName())[1], false);
+          Utilities.getDbTableName(alterTable.getDbTableName())[1], false);
```

Comment:
I think we might want to mark Utilities.getDbTableName as deprecated or :dagger: it
can be done in a followup as well

Reply:
I've marked it deprecated this time.
Some places rely on some of the state changes from this, so killing it in 1 patch would have been harder.